thirdeye/thirdeye-pinot/src/test/resources/org/apache/pinot/thirdeye/detection/yaml/compositePipelineTranslatorTestResult-1.json

```diff
@@ -1,48 +1,73 @@
 {
   "properties": {
-    "className": "org.apache.pinot.thirdeye.detection.wrapper.ChildKeepingMergeWrapper",
-    "maxGap": 0,
-    "nested": [{
-      "nestedMetricUrns": ["thirdeye:metric:1:D1%3Dv1:D1%3Dv2:D2%3Dv3"],
-      "className": "org.apache.pinot.thirdeye.detection.algorithm.DimensionWrapper",
-      "metricUrn": "thirdeye:metric:1:D1%3Dv1:D1%3Dv2:D2%3Dv3",
-      "nested": [{
-        "filter": "$thresholdFilter_2:THRESHOLD_RULE_FILTER",
-        "className": "org.apache.pinot.thirdeye.detection.wrapper.AnomalyFilterWrapper",
-        "nested": [{
-          "filter": "$thresholdFilter_1:THRESHOLD_RULE_FILTER",
-          "className": "org.apache.pinot.thirdeye.detection.wrapper.AnomalyFilterWrapper",
-          "nested": [{
-            "baselineValueProvider": "$maxThreshold_1:THRESHOLD",
-            "className": "org.apache.pinot.thirdeye.detection.wrapper.BaselineFillingMergeWrapper",
-            "maxGap": 0,
-            "nested": [{
-              "bucketPeriod": "P1D",
-              "className": "org.apache.pinot.thirdeye.detection.wrapper.AnomalyDetectorWrapper"
-            }],
-            "detector": "$maxThreshold_1:THRESHOLD",
-            "maxDuration": 100
-          }]
-        }]
-      }, {
-        "filter": "$thresholdFilter_3:THRESHOLD_RULE_FILTER",
-        "className": "org.apache.pinot.thirdeye.detection.wrapper.AnomalyFilterWrapper",
-        "nested": [{
-          "baselineValueProvider": "$maxThreshold_2:THRESHOLD",
-          "className": "org.apache.pinot.thirdeye.detection.wrapper.BaselineFillingMergeWrapper",
-          "maxGap": 0,
-          "nested": [{
-            "bucketPeriod": "P1D",
-            "className": "org.apache.pinot.thirdeye.detection.wrapper.AnomalyDetectorWrapper"
-          }],
-          "detector": "$maxThreshold_2:THRESHOLD",
-          "maxDuration": 100
-        }]
-      }],
-      "minContribution": 0.05,
-      "dimensions": ["D1", "D2"]
-    }],
-    "maxDuration": 100
+    "className": "org.apache.pinot.thirdeye.detection.wrapper.GrouperWrapper",
+    "grouper": "$test_grouper:MOCK_GROUPER",
+    "nested": [
+      {
+        "className": "org.apache.pinot.thirdeye.detection.wrapper.ChildKeepingMergeWrapper",
+        "maxGap": 0,
+        "nested": [
+          {
+            "nestedMetricUrns": [
+              "thirdeye:metric:1:D1%3Dv1:D1%3Dv2:D2%3Dv3"
+            ],
+            "className": "org.apache.pinot.thirdeye.detection.algorithm.DimensionWrapper",
```

Comment:
Would you please give some examples that "grouper" is configured?

Reply:
You can take a look at the `pipeline-config-3.yaml` file in this PR for an example of how grouper can be configured.